src/i18n/content.json

```diff
@@ -31,6 +31,10 @@
           {
             "name": "codepen",
             "link": "https://codepen.io/iamkun/pen/YzWMaVr"
+          },
+					{
+            "name": "CodeSandbox(SPA)",
```

Comment:
can we use the name `codesandbox` to keep the same with others?

Reply:
YES.